### PR TITLE
add property IsCommutative

### PR DESCRIPTION
**Pull request by heiderich**

### CODE REVIEW

File: gap/Bialgebroids.gi [hunk 1]
```diff
@@ -833,6 +833,29 @@ InstallMethod( AddAntipode,
...
+    local arrows, is_commutative, x, y;
+    
+    arrows := Arrows( UnderlyingQuiverAlgebra( A ) );
+    
+    is_commutative := true;
+
+    for x in arrows do
+      for y in arrows do
+        if not x*y = y*x then
+          is_commutative := false;
```
Comment (mohamed-barakat) [inline gap/Bialgebroids.gi]: return false;

File: gap/Bialgebroids.gi [hunk 2]
```diff
@@ -833,6 +833,29 @@ InstallMethod( AddAntipode,
...
+        fi;
+      od;
+    od;
+    
+    return is_commutative;
```
Comment (mohamed-barakat) [inline gap/Bialgebroids.gi]: return true;

File: gap/Bialgebroids.gi [hunk 3]
```diff
@@ -833,6 +833,29 @@ InstallMethod( AddAntipode,
...
+     
+  function( A )
+    local arrows, is_commutative, x, y;
+    
+    arrows := Arrows( UnderlyingQuiverAlgebra( A ) );
+    
+    is_commutative := true;
+
+    for x in arrows do
+      for y in arrows do
```
Comment (mohamed-barakat) [inline gap/Bialgebroids.gi]: This code is far from optimal. You are checking the whole square, whereas you only need to check the upper/lower triangle (excluding the diagonal).

**Comment by heiderich** [comment]
I addresses all comments in e311098, which I force-pushed.

**Comment by mohamed-barakat** [comment]
Looks good :)

**Pull request merged**